CONTRIBUTING.md

```diff
@@ -155,3 +209,5 @@ Heavily adapted from the contributing files from the
 [Puppet project](https://github.com/puppetlabs/puppet),
 [Factory Girl Rails](https://github.com/thoughtbot/factory_girl_rails/blob/master/CONTRIBUTING.md),
 and [Idris](https://github.com/idris-lang/Idris-dev).
+The section on reviewing code is partly based on
+[lornajane's blog entry](https://lornajane.net/posts/2015/code-reviews-before-you-even-run-the-code).
```

Comment:
This would be redundant if you mention it in the section as suggested above, no?